apidoc/v2/openapi.yaml

```diff
@@ -5430,3 +5436,175 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/extra_fields_keys'
+  # DSPACE
+  /dspace:
+    summary: Interact with the connected DSpace repository.
+
+    get:
+      tags: ['Dspace']
+      summary: Read DSpace information.
+      operationId: dspace-read
+      parameters:
+        - name: dspace_action
+          in: query
+          description: |
+            Action to perform:
+              - `GetCollections`: list available DSpace collections.
+              - `GetTypes`: list DSpace item types (submission vocabularies).
+            If omitted, the endpoint returns an empty array.
+          required: false
+          schema:
+            type: string
+            enum: [ 'getcollections', 'gettypes' ]
+      responses:
+        '200':
+          description: |
+            Result depends on the `dspace_action` parameter:
+              - `GetCollections`: list of collections.
+              - `GetTypes`: list of vocabulary entries.
+              - No action: empty array.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: array
+                    description: Empty array when no action is given.
+                    items: {}
+                  - type: array
+                    description: Collections array returned by DSpace.
```

Comment:
can we get a permalink in the description to dspace api doc for this? Because we are referencing a reply from an external service, it's nice to point to their doc too